edb/lib/cfg.edgeql

```diff
@@ -23,6 +23,7 @@ CREATE ABSTRACT INHERITABLE ANNOTATION cfg::backend_setting;
 CREATE ABSTRACT INHERITABLE ANNOTATION cfg::internal;
 CREATE ABSTRACT INHERITABLE ANNOTATION cfg::requires_restart;
 CREATE ABSTRACT INHERITABLE ANNOTATION cfg::system;
+CREATE ABSTRACT INHERITABLE ANNOTATION cfg::compilation;
```

Comment:
I don't quite like the name, it's gonna be confusing in introspection :/ Maybe `cfg::required_for_compiler`?

Reply:
How about `affects_compilation`?